app/javascript/dashboard/routes/dashboard/conversation/contact/ContactForm.vue

```diff
@@ -172,6 +206,11 @@ export default {
     },
     bio: {},
   },
+  computed: {
```

Comment:
This code can be removed.

Reply:
@nithindavid If I remove it from the computed property, the values will not render in multi-select.

Reply:
@iamsivin We can initialise the value in `data`.